azure-pipelines.yml

```diff
@@ -396,14 +396,13 @@ stages:
       - script: eng/validate-rules-missing-documentation.cmd -ci
         displayName: Validate rules missing documentation
 
-      - task: PowerShell@2
+      - pwsh: |
```

Comment:
Does that use a `pwsh` that is controlled by AzDO or shell out to the `pwsh` that we installed via local tools?

Reply:
pwsh is the Powershell task with `pwsh` set to true.